devito/function.py

```diff
@@ -815,6 +781,66 @@ def __indices_setup__(cls, **kwargs):
     def __shape_setup__(cls, **kwargs):
         return kwargs.get('shape', (kwargs.get('npoint'),))
 
+    def _arg_defaults(self, alias=None):
+        """
+        Returns a map of default argument values defined by this symbol.
+
+        :param alias: (Optional) name under which to store values.
+        """
+        key = alias or self
+        args = super(AbstractSparseFunction, self)._arg_defaults(alias=alias)
+        for child in self.child_functions:
+            args.update(child._arg_defaults(alias=getattr(key, child.name).name))
```

Comment:
`alias` should be `gettattr` not just the name.

Reply:
Fixed